### PR TITLE
Add licensebadge to stories

### LINKED ISSUE #63: Add LicenseIconography to Vocabulary
**Description**
We want to add the LicenseIconography element, currently in Vue Vocabulary, to Vocabulary.
https://cc-vue-vocabulary.netlify.com/storybook/?path=/story/elements-licenseiconography--creative-commons

Once we have added this element and associated styles into Vocabulary, we'd like to do the following:

1. Break each icon into individual Styles, to match the Downloads page:
https://creativecommons.org/about/downloads
2. Add the LicenseBadge (within Patterns) from Vue Vocabulary as a page in License Iconography in Vocabulary.

That is, the new License Iconography page in Vocabulary should contain the elements from License Iconography in Vue Vocabulary, as well as the License Badge patterns from Vue Vocabulary.

### PR DESCRIPTION
**Pull request by Dhruvi16**
**Fixes**

Partially addresses #63 

**Description**

This adds license badges both the normal and smaller version to the story. Badges are added in the list form under the name of LicenseBadge. The SVGs are added to the assets->license_badge->big/small.

**Type of PR** 
This PR is a **feature**.

**Screenshots**

image

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
@Dhruvi16 I've made some changes that not only fix the missing badges but also reorganise the assets to account for the [smaller 80x15 license badges](https://creativecommons.org/about/downloads) that you will hopefully add in the future.

**Comment by Dhruvi16** [comment]
Hey! I tried adding id too. It was not working for me then but it is now!

**Comment by Dhruvi16** [comment]
I am not sure if this is a feature!

**Comment by dhruvkb** [comment]
@Dhruvi16 it is very much a feature!

File: src/stories/license_badge.stories.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  '<svg xmlns="http://www.w3.org/2000/svg">' +
+  '<use href="license_badges/big/by.svg#layer1"></use>' +
+  '</svg>'
+export const BY = () => `<div>${by}</div>`
+
+const bysa = '' +
+  '<svg xmlns="http://www.w3.org/2000/svg">' +
+  '<use href="license_badges/big/by_sa.svg#layer1"></use>' +
+  '</svg>'
+export const BYSA = () => `<div>${bysa}</div>`
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.js]: Use _ to separate words like BY_SA. That underscore changes to a space when it's rendered in the storybook.

File: src/stories/license_badge.stories.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+const by = '' +
+  '<svg xmlns="http://www.w3.org/2000/svg">' +
+  '<use href="license_badges/big/by.svg#layer1"></use>' +
+  '</svg>'
+export const BY = () => `<div>${by}</div>`
+
+const bysa = '' +
+  '<svg xmlns="http://www.w3.org/2000/svg">' +
+  '<use href="license_badges/big/by_sa.svg#layer1"></use>' +
+  '</svg>'
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.js]: Make this a function like so:
```js
const licenseSVG = (size, license) => `<svg ...>
<use href="licensebadges/${size}/${license}.svg#layer1"></use>
</svg>`
```

File: src/stories/license_badge.stories.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  '<svg xmlns="http://www.w3.org/2000/svg">' +
+  '<use href="license_badges/small/by.svg#layer1"></use>' +
+  '</svg>'
+export const BYSmall = () => `<div>${bySmall}</div>`
+
+const bysaSmall = '' +
+  '<svg xmlns="http://www.w3.org/2000/svg">' +
+  '<use href="license_badges/small/by_sa.svg#layer1"></use>' +
+  '</svg>'
+export const BYSASmall = () => `<div>${bysaSmall}</div>`
```
Comment (dhruvkb) [inline src/stories/license_badge.stories.js]: With the above feedback this becomes BY_SA_Small.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Some changes for reducing code duplication and improving readability.

File: src/stories/license_badge.stories.js (hunk 2 above)
Comment (Dhruvi16) [reply src/stories/license_badge.stories.js]: This is great for code duplication. I should have thought about it. But this fails in the case of cc_zero and public domain svgs where the id is` #Layer1`. I will think of something and get back to you.

Comment (Dhruvi16) [reply src/stories/license_badge.stories.js]: One idea off the top of my head is to create id a variable and pass it through the parameter. But IMO parameters should be more generalized and be relevant to the characteristics of the entity.
Do you think we should export id as a parameter here?

Comment (dhruvkb) [reply src/stories/license_badge.stories.js]: You can edit the SVGs and make the IDs same for all of them.

**Comment by Dhruvi16** [comment]
I am getting a Lint error - 
image

This can be fixed by replacing BY_Small with BY_SMALL or BYSmall. I think we BY_SMALL. Or we can disable camel case for these lines.

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**